examples/jsm/renderers/webgl/nodes/WebGLNodeBuilder.js

```diff
@@ -533,7 +533,7 @@ class WebGLNodeBuilder extends NodeBuilder {
 
 	getInstanceIndex() {
 
-		const node = ( viewportCoordinate.y ).mul( viewportResolution.width ).add( viewportCoordinate.x );
+		const node = ( viewportCoordinate.y ).mul( this.outComputeBuffer.texture.image.width ).add( viewportCoordinate.x );
```

Comment:
Don't know why `viewportResolution` works incorrectly...

Reply:
This should be bound to be instance related as well.